### ISSUE TITLE
Error running on Ubuntu (Parallels)

### ISSUE THREAD

**Issue by EmberManYT** [perfectly-balanced/PathBench#78]
I'm completely new to this, and I'm getting an error trying to run this on Ubuntu 22.04 on a Mac through Parallels.
sklearn was deprecated so I installed scikit-learn, which is the only change I made

python3 src/main.py -v gives the following:

```
line 128, in _get_shape
    raise InvalidArgumentsError(
nptyping. error. InvalidArgumentsError: Unexpected argument '(typing. Any, typing.A ny, typing.Any)', expecting Shape[<ShapeExpression>] or Literal[<shapeExpression
>] or typing.Any.
```

**Comment by Yikic** [comment]
maybe you CAN try  'vim'

**Comment by lijingze-counter** [comment]
I directly used the deprecated sklearn and the same problem occurred. Has your problem been solved?

nptyping.error.InvalidArgumentsError: Unexpected argument '(typing.Any, typing.Any, typing.Any)', expecting Shape[<ShapeExpression>] or Literal[<ShapeExpression>] or typing.Any.

**Comment by Yikic** [comment]
yeah,the sklearn package has deprecated,you can pip install scikit-learn instead

**Comment by fernandesdominic22** [comment]
Hi  @uncobruce, I'm using python 3.8.10 on Ubuntu 20.04. I get this error when running python3 src/main.py -v:
 
 ```
line 128, in _get_shape
    raise InvalidArgumentsError(
nptyping.error.InvalidArgumentsError: Unexpected argument '(typing.Any, typing.Any, typing.Any)', expecting Shape[<ShapeExpression>] or Literal[<ShapeExpression>] or typing.Any.
```

I would greatly appreciate any help. Thanks
